src/Resources/skeleton/Class.tpl.php

```diff
@@ -2,6 +2,10 @@
 
 namespace <?= $namespace; ?>;
 
+/**
+ * Class <?= $class_name ?>
+ * @package <?= $namespace; ?>
+ */
```

Comment:
These comments don’t add any value - tools already know the class and package - so we should not add these :)